pymatgen/analysis/defects/core.py

```diff
@@ -29,13 +28,15 @@
 
 logger = logging.getLogger(__name__)
 
+kb = 8.6173324e-5  # eV / K
```

Comment:
Add this to `pymatgen.core.units` if it's not already there and import from there.

Reply:
Done, the pylint caught some problems with the defect utils code.  Should be fixed now.

Reply:
Should we have a pre-commit-config.yaml for pymatgen?  I think currently the precommit only does pycodestyle

Reply:
Sure, as long as we have some description in the docs to explain what it does and how to use it. For people who do not have pre-commit installed it's ignored correct?

Reply:
> For people who do not have pre-commit installed it's ignored correct?
Correct